rc/base/git.kak

```diff
@@ -14,8 +18,16 @@ hook -group git-commit-highlight global WinSetOption filetype=git-commit %{
     add-highlighter window/git-commit-highlight/comments/ regex "\b(?:(modified)|(deleted)|(new file)|(renamed|copied)):([^\n]*)$" 1:yellow 2:red 3:green 4:blue 5:magenta
 }
 
+hook -group git-commit-highlight global WinSetOption filetype=git-notes %{
+    add-highlighter window/git-notes-highlight regions
+    add-highlighter window/git-notes-highlight/comments region '^\h*#' '$' group
```

Comment:
why make that a group if you only put a fill highlighter inside ?

Reply:
Because there's more data within the comments that could be highlighted similarly to how the commit highlighter does it, I thought somebody would mention it. If we only care about highlighting full comments, I guess we don't need to leave that door open.